src/pp.js

```diff
@@ -131,16 +131,26 @@ function hasHardLine(doc) {
   });
 }
 
-function _makeIndent(n) {
+function _makeIndent(tabWidth) {
   var s = "";
 
-  for (var i = 0; i < n; i++) {
+  for (var i = 0; i < tabWidth; i++) {
     s += " ";
   }
 
   return s;
 }
 
+function _repeatIndent(n, indent) {
```

Comment:
Those two functions are doing the same thing. Could you rename this one to _repeatString and replace the call sites of `_makeIndent(n)` to `_repeatString(n, " ")`